nvflare/__init__.py

```diff
@@ -15,5 +15,3 @@
 from . import _version
 
 __version__ = _version.get_versions()["version"]
-
-from nvflare.private.fed.app.simulator.simulator_runner import SimulatorRunner
```

Comment:
we might want to keep this line if we want people to be able to 
`import SimulatorRunner from nvflare`

Reply:
There was an error previously, that's why removed it temporarily. Now the error has been fixed. Added this back.